src/components/internals/create-select-styles.js

```diff
@@ -217,6 +217,9 @@ const valueContainerStyles = () => (base) => ({
   padding: '0',
   backgroundColor: 'none',
   overflow: 'hidden',
+  '& svg': {
+    flexShrink: 0,
```

Comment:
This is to prevent the SVGs from shrinking when there's not enough horizontal space because they're inside a `Spacings.Inline`